transactions/src/main/java/org/ehcache/transactions/xa/configuration/XAStoreConfiguration.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.transactions.xa.configuration;
+
+import org.ehcache.spi.service.ServiceConfiguration;
+import org.ehcache.transactions.xa.XAStore;
+
+/**
+ * @author Ludovic Orban
+ */
+public class XAStoreConfiguration implements ServiceConfiguration<XAStore.Provider> {
+
+  private final String uniqueXAResourceId;
+
+  //TODO: ideally, the uniqueXAResourceIdSuffix should be the cache alias, injected by the CM
```

Comment:
First: uniqueXAResourceIdSuffix should read uniqueXAResourceId, I'll fix that

Then, this can be discussed; we may still want the unique XAResource ID to default to the cache name at a later stage, or close that door forever.